### PR TITLE
scrum-2673 add citation to search

### PR DESCRIPTION
**Pull request by FB-ping**
add citation to search bar

### CODE REVIEW

**Review by azurebrd — COMMENTED** [review]
This looks good to me, but I don't know how the timing of merging and deploying the API and UI are, so I'll defer to @valearna

**Comment by FB-ping** [comment]
another PR for agr_literture_service should be reviewed first.

**Comment by valearna** [comment]
I'll merge this PR after restarting the API on stage

**Review by valearna — APPROVED** [review]

**Pull request merged**
